### PR TITLE
Add option to scan multiple assemblies for Automapper profiles

### PR DESCRIPTION
**Pull request by pmotameni**
Add support to read multiple assemblies for Automapper profiles. 
Plus a cleanup

### CODE REVIEW

File: src/BlackSlope.Api/Startup.cs [hunk 1]
```diff
@@ -85,17 +85,8 @@ private void CorsConfiguration(IServiceCollection services)
...
-                options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme;
-            })
-            .AddJwtBearer(options =>
-            {
-                options.Authority = string.Format(HostConfig.AzureAd.AadInstance, HostConfig.AzureAd.Tenant);
-                options.Audience = HostConfig.AzureAd.Audience;
-            });
-        }
+        // make a list of projects in the solution which must be scanned for mapper profiles
+        private static string[] GetAssemblyNamesToScaneForMapperProfiles() =>
```
Comment (chelium) [inline src/BlackSlope.Api/Startup.cs]: Think there's an extra 'e' after Scan

Comment (pmotameni) [reply src/BlackSlope.Api/Startup.cs]: Fixed

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -78,21 +77,22 @@ public static class BlackSlopeServiceCollectionExtensions
...
 
         /// <summary>
-        /// Add AutoMapper service to the Service Collection and configure it
+        /// Add AutoMapper service to the Service Collection and configure it assemblies 
         /// </summary>
         /// <param name="services"></param>
+        /// <param name="assemblyNamesToScan"></param>
         /// <returns></returns>
-        public static IServiceCollection AddAutoMapper(this IServiceCollection services)
+        public static IServiceCollection AddAutoMapper(this IServiceCollection services, string[] assemblyNamesToScan)
```
Comment (andyk1989) [inline src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: nit: `IEnumerable<string> assemblyNamesToScan`

File: src/BlackSlope.Api/Startup.cs [hunk 3]
```diff
@@ -85,17 +85,8 @@ private void CorsConfiguration(IServiceCollection services)
             });
         }
 
-        private void AuthenticationConfiguration(IServiceCollection services)
```
Comment (andyk1989) [inline src/BlackSlope.Api/Startup.cs]: This was the cleanup piece right?

Comment (pmotameni) [reply src/BlackSlope.Api/Startup.cs]: yes, it has been moved into common as AddAzureAd extension method for ServiceCollection.

File: src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs (hunk 2 above)
Comment (pmotameni) [reply src/BlackSlope.Api.Common/Extensions/BlackSlopeServiceCollectionExtensions.cs]: Updated

**Pull request merged**